nodejs/week1/meals/src/backend/routes/meal.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const app = express();
```

Comment:
Remove. Same explanation than the previous file. Also export only router function.